pkg/client/client.go

```diff
@@ -297,3 +297,16 @@ func DefaultOpenShiftUserAgent() string {
 	version = seg[0]
 	return fmt.Sprintf("%s/%s (%s/%s) openshift/%s", path.Base(os.Args[0]), version, runtime.GOOS, runtime.GOARCH, commit)
 }
+
+type HybridClient struct {
```

Comment:
Have a look at our dc scaler and reaper implementations. They use both of the client under the hood w/o exposing a hybrid client.